### DISCUSSION TITLE
Running into vector error on the CPP Object Detector example

### DISCUSSION THREAD

**Discussion by louispeterson22** [AxisCommunications/acap-computer-vision-sdk-examples#47] [General] [answered]
Using a P3265, I am running into an error related to an array while going through the C++ object detector example:

![cpp error](https://user-images.githubusercontent.com/59744377/174888709-16b1bee4-6f31-4bba-8d5f-76f2c882026d.png)

When we run the python version of it, we receive a similar/same error as well:

![python error](https://user-images.githubusercontent.com/59744377/174888723-038cdaa6-bcf7-4558-b09c-f25e7c64deea.png)

Any suggests would be greatly appreciated!

**Comment by Corallo** [comment, marked as answer]
Hello @louispeterson22 

Are you using a custom model or the one proposed in the example?
Which firmware version are you using?

**Reply by louispeterson22** [reply]
Hello,

We are using the one proposed in the example.  I see the firmware was a pre-release version at CVP-21.41.4_beta.  We updated to 10.11.76 and that error is gone, but we receive a gRPC call return code: Connect Failed message 

![MicrosoftTeams-image](https://user-images.githubusercontent.com/59744377/175068811-517a1a14-d1a0-40cf-a909-f1db885822df.png)
o

**Reply by Corallo** [reply]
The latest firmware (10.11) have a problem that makes the inference server to crash.
https://github.com/AxisCommunications/acap-computer-vision-sdk-examples/issues/28

We reccomand to step back to 10.10 to avoid this issue

**Comment by louispeterson22** [comment]
Thank you so much, that worked!
